includes/class-press-sync.php

```diff
@@ -414,8 +424,9 @@ public function count_objects_to_sync( $objects_to_sync ) {
 
 		global $wpdb;
 
-		$where_clause = '';
-		$where_clause = $this->get_synced_object_clause( $objects_to_sync );
+		$where_clause  = '';
+		$where_clause  = $this->get_synced_object_clause( $objects_to_sync );
```

Comment:
This line just overwrites the previous line. Is the previous line needed?

Reply:
tbh I wasn't really paying attention to the lines here and above, it's just marked "changed" from indentation. Will note that for the future, however.